community/io/src/main/java/org/neo4j/io/fs/FileSystemAbstraction.java

```diff
@@ -92,7 +92,7 @@
 
     void truncate( File path, long size ) throws IOException;
 
-    long lastModifiedTime( File file ) throws IOException;
+    long lastModifiedTime( File file );
```

Comment:
Not part of this PR, but we encourage people to run with `noatime` (https://neo4j.com/docs/operations-manual/current/performance/linux-file-system-tuning/) resulting in this method being very dangerous to rely on. A javadoc comment could be added here.

Reply:
It's something I looked at, but ended up not using anyway. I kept it because it aligns behaviour that was inconsistent.